page-objects/pages/administrationPages/ADMNSTRTNAdministerStudentsPage.js

```diff
@@ -18,13 +18,33 @@ const addStudentSubmitBtn = 'div.modal.fade.add-modal.in button.btn-submit';
 const passwordInput = '#passwd';
 const createBirthday = '#birthday';
 const sendConsentFormEmails = '.btn-send-links-emails';
+const editStudentBtn = 'a[title="Nutzer bearbeiten"]';
```

Comment:
That's not what I meant: 
```suggestion
please use
const editStudentBtn = '.table-actions .btn .fa-edit';
```